test/common/markdown/markdown.tests.ps1

```diff
@@ -17,7 +17,7 @@ Describe 'Common Tests - Validate Markdown Files' -Tag 'CI' {
         {
             $NpmInstalled = "Installed"
             Write-Verbose -Message "NPM is checking Gulp is installed. This may take a few moments." -Verbose
-            start-nativeExecution { npm install --silent }
+            start-nativeExecution { yarn }
             start-nativeExecution { npm install 'gulp@4.0.0' --silent }
```

Comment:
If we are going to use yarn, can we use the yarn commands?

Reply:
line 25 too

Reply:
Good point.  Not sure why this is here anyway as it should be installed by package.json.

Reply:
It should... I added it because it wasn't always installed.

Reply:
Replaced it with yarn add.